pkg/cmd/server/apis/config/types.go

```diff
@@ -399,6 +399,9 @@ type MasterConfig struct {
 	EtcdConfig *EtcdConfig
 	// OAuthConfig, if present start the /oauth endpoint in this process
 	OAuthConfig *OAuthConfig
+	// WebhookTokenAuthnConfig, if present configures remote token reviewers
+	WebhookTokenAuthenticators []WebhookTokenAuthenticators
```

Comment:
probably belongs under MasterAuthConfig

Reply:
do we have a use case for allowing more than one, or is that just future-proofing?

Reply:
Maybe seem that it is more similar to OAuthConfig than RequestHeaderAuthenticationOptions as an auth type, why do we not have OAuthConfig under MasterauthConfig ?

Reply:
OAuth component is distinct from the master API. OAuth config is for the OAuth server. MasterAuthConfig is for the API server.

Reply:
> Maybe seem that it is more similar to OAuthConfig than RequestHeaderAuthenticationOptions as an auth type

RequestHeaderAuthenticationOptions and WebhookTokenAuthenticators both mirror upstream API authentication configuration options. We should group them under MasterAuthConfig.

Reply:
Ok, makes sense.